pkg/builder/build.go

```diff
@@ -114,12 +131,10 @@ func (b *Builder) Build(r reconcile.Reconciler) (manager.Manager, error) {
 		return nil, err
 	}
 
-	a := &application{mgr: b.mgr, ctrl: b.ctrl}
-
 	// Reconcile type
 	s := &source.Kind{Type: b.apiType}
```

Comment:
blech, we should fix these single-letter variables in a follow-up

Reply:
done